glide.yaml

```diff
@@ -0,0 +1,4 @@
+package: github.com/rigelrozanski/breaking
```

Comment:
hmmmm. with Golang, you'll want to `go get` the original repo (always) then add a remote that is your fork, rather than `git clone` (or `go get`) your fork